data/human/free worlds 0 prologue.txt

```diff
@@ -1707,16 +1903,30 @@ mission "FW Bounty 2"
 	
 	on offer
 		conversation
-			`A Free Worlds officer waves to you as you are walking through the spaceport. "Hello Captain <last>," he says. "I've heard you've helped us in the past to deal with pirate threats. Interested in another bounty-hunting mission?"`
+			`A Free Worlds officer waves to you as you are walking through the spaceport. "Hello Captain <last>," he says. "I've heard you've helped us in the past to deal with pirate threats. Interested in another bounty hunting mission?"`
 			choice
+				`	"What will it be this time?"`
 				`	"Sure, I do love destroying pirates!"`
+					goto accept
 				`	"Sorry, I'm way too busy right now."`
 					defer
-			`	He grins. "Your target this time is a retrofitted heavy freighter by the name of Moonless Night that has been stealing large amounts of cargo from our supply convoys. It was last seen in the Delta Sagittarii system. Good luck, Captain."`
+			
+			`	"There's a retrofitted heavy freighter by the name of Moonless Night that has been stealing large amounts of cargo from our supply convoys. It was last seen in the Delta Sagittarii system. Be warned that this ship is a tough one, fitted with more heavy lasers than I've ever seen anyone fit on a single ship, but it's slow as a sloth, so you might be able to hit it from afar. The bounty on it is <payment>."`
```

Comment:
What's so special about this mission that the <npc> can't be used?

Reply:
Nothing. It can be used and just wasn't.

Reply:
Ah, ok.